data/json/monster_weakpoints/generic_weakpoints.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "type": "weakpoint_set",
-    "id": "wps_natural_armor",
+    "id": "wps_armored_hide",
```

Comment:
Please revert your ID changes. You can add a comment to the entries bellow the ID to clarify if you want but changing the ID's needlessly breaks 3rd party mods.

Otherwise at least to my eyes this is good to merge.

Reply:
Latest commit reverted this id change.

Reply:
all good, you are also allowed to just take some time on these things.